dist/fedora-infra/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fedora:30
+
+# build: project sources
+ADD . /src
+WORKDIR /src
+
+# build: system utilities and libraries
+RUN dnf -y install g++ openssl-devel
+
+# build: pinned Rust stable toolchain (only use in case of temporary distro breakage)
+#ADD https://static.rust-lang.org/dist/rust-1.35.0-x86_64-unknown-linux-gnu.tar.gz rust.tar.gz
+#RUN tar -xf rust.tar.gz --strip 1 && ./install.sh
+
+# build: system Rust toolchain
+RUN dnf -y install rust cargo
+
+# build: graph-builder binary
+RUN cargo build --release --bin graph-builder && mv /src/target/release/graph-builder /usr/local/bin/
+
+# build: policy-engine binary
+RUN cargo build --release --bin policy-engine && mv /src/target/release/policy-engine /usr/local/bin/
+
+# build: cleanup
+RUN cargo clean
```

Comment:
If you want to clean up you could remove the source directory as a whole, as well as `$HOME/.cargo` and removing the toolchain.

Reply:
Ack, I made this more aggressive as you suggested. I'm just leaving the RPMs around at this point.